examples/data-objects/prosemirror/src/index.ts

```diff
@@ -25,20 +25,20 @@ const defaultComponentId = "default";
 const smde = new ProseMirrorFactory();
 
 class ProseMirrorRuntimeFactory extends RuntimeFactoryHelper {
-	public async instantiateFirstTime(runtime: ContainerRuntime): Promise<void> {
+	public async instantiateFirstTime(runtime: IContainerRuntime): Promise<void> {
```

Comment:
Not directly relevant to this PR, but I'm not sure if anyone uses RuntimeFactoryHelper in the wild, maybe we consider making it internal longer term?  IMO we should get away from providing superclasses in general, and this one in particular doesn't seem to do a whole lot.

Reply:
the only complexity will that be that it is the base class for BaseContainerRuntimeFactory which is the base class for ContainerRuntimeFactoryWithDefaultDataStore in aqueduct.